recipes/slugify/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "slugify" %}
+{% set version = "0.0.1" %}
+{% set md5 = "557a7e401ea3c9716d29389c56cf82c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
```

Comment:
Due to a Windows/Python 3 issue with the ebooklib dependency, we skip the Windows/Python 3 builds for now.